<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMacros</class>
 <widget class="QDialog" name="DialogMacros">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Macro Commands</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="buttonNew">
           <property name="text">
            <string>New</string>
           </property>
           <property name="icon">
            <iconset theme="document-new">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonDelete">
           <property name="text">
            <string>Delete</string>
           </property>
           <property name="icon">
            <iconset theme="edit-delete">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonCopy">
           <property name="text">
            <string>Copy</string>
           </property>
           <property name="icon">
            <iconset theme="edit-copy">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonUp">
           <property name="text">
            <string>Move</string>
           </property>
           <property name="icon">
            <iconset theme="go-up">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonDown">
           <property name="text">
            <string>Move</string>
           </property>
           <property name="icon">
            <iconset theme="go-down">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="listItems"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Select a macro menu item from the list at left.
Select &quot;New&quot; to add a new command to the menu.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Menu Entr&amp;y</string>
           </property>
           <property name="buddy">
            <cstring>editName</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>(&gt; for sub-menu, @ language mode)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="editName"/>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="KeySequenceEdit" name="editAccelerator"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Acce&amp;lerator</string>
           </property>
           <property name="buddy">
            <cstring>editAccelerator</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1" rowspan="2">
          <widget class="QPushButton" name="buttonPasteLRMacro">
           <property name="text">
            <string>&amp;Paste Learn/
Replay Macro</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkRequiresSelection">
         <property name="text">
          <string>&amp;Requires Selection</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Macro Command to E&amp;xecute</string>
     </property>
     <property name="buddy">
      <cstring>editMacro</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="editMacro">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="buttonOK">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-ok">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonApply">
       <property name="text">
        <string>&amp;Apply</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-ok-apply">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCheck">
       <property name="text">
        <string>&amp;Check</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClose">
       <property name="text">
        <string>&amp;Close</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-close">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KeySequenceEdit</class>
   <extends>QLineEdit</extends>
   <header>KeySequenceEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonClose</sender>
   <signal>clicked()</signal>
   <receiver>DialogMacros</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>578</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>475</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
